actioncable/CHANGELOG.md

```diff
@@ -1,2 +1,17 @@
+*   Add ability to detect a half-open connection
+
+    ActionCable doesn't have a mechanism to detect if it's connection is
+    half-open (if there isn't a client listening on the other end).
+
+    A half-open connectionn can linger around for up to half an hour using
```

Comment:
Typo: connection

Reply:
Fixed in the CHANGELOG file and the commit message. Thank you!